src/fsharp/FSharp.Core/FSharp.Core.fsproj

```diff
@@ -211,9 +212,41 @@
                                                 '$(TargetFramework)' == 'XNA\5.0' OR 
                                                 '$(TargetFramework)' == 'sl3-wp' "/>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)' == 'coreclr'">
+    <Reference Include="$(FSharpSourcesRoot)\..\packages\System.Collections\4.0.10-beta-22809\ref\any\System.Collections.dll" />
```

Comment:
@KevinRansom that sounds reasonable.  Just wanted to make sure that was the goal.